content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: 'Prisma Client extensions'
+metaTitle: 'Prisma Client extensions (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+preview: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+Use Prisma Client extensions to add functionality to the models in your schema, results objects, queries, or Prisma Client itself.
```

Comment:
Is this really "models in your schema" and not more "models created in Client based on the models in your schema"?

Reply:
Yes, I suppose technically that's correct, and we want to be technically correct. For the sake of brevity, I think "your models" will probably suffice.